package.json

```diff
@@ -22,13 +22,15 @@
     "@babel/plugin-transform-runtime": "^7.4.3",
     "@babel/preset-env": "^7.0.0",
     "@babel/runtime": "^7.4.3",
+    "auth0-js": "^9.10.2",
     "autoprefixer": "^9.4.2",
     "axios": "^0.18.0",
     "babel-loader": "^8.0.0",
     "css-loader": "^0.28.11",
     "es6-promise": "^4.2.4",
     "fast-async": "7",
     "file-loader": "^1.1.11",
+    "fontfaceobserver": "^2.1.0",
```

Comment:
I like the idea of having this included this way, but why do we have it in `font_load.html` and here? Maybe I'm misunderstanding how that works.

Reply:
It's really just for documentation's sake. To be aware of what version we are using. There's no direct connection between what's inlined in `font_load.html` and what lives in `node_modules` after a `yarn`.